doc: Thingy user guide update

umapraseeda opened the pull request:
Move the infocenter GSG to NCS.
Also, removed some sections from Working with Thingy
user guide as its already covered by the GSG.
Updated the screenshots.
TECHDOC-2144

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-6561
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+#. Update the nRF52840 :term:`System on Chip (SoC)` application:
+
+   a. Switch off the Thingy:91.
+   #. Press **SW4** while switching **SW1** to the ON position.
```
The ON position should be in bold as in the Preparing for setup section.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+#. Update the modem firmware on the nRF9160 System in Package (SiP):
+
+   a. Switch off the Thingy:91.
+   #. Press **SW3** while switching **SW1** to the ON position.
```
The ON position should be in bold as in the Preparing for setup section.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+#. Update the nRF9160 SiP application:
+
+   a. Switch off the Thingy:91.
+   #. Press **SW3** while switching **SW1** to the ON position.
```
The ON position should be in bold as in the Preparing for setup section.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+
+4. Update the modem firmware on the nRF9160 System in Package (SiP):
+
+   a. Set the SWD selection switch **SW2** to nRF91.
```
The switch value nRF91 should be in bold like in updating the 840 app.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+
+5. Update the nRF9160 SiP application:
+
+   a. Make sure the SWD selection switch **SW2** is set to nRF91.
```
The switch value nRF91 should be in bold.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+
+..
+
+Next, you need to create an nRF Cloud account if you do not have one already.
```
I'm not sure of this is needed. These comments at the ends of topics were added to the HTML version because the Infocenter structure is not as linear as a PDF or NCS page and it's not as obvious what you need to do next.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+#. Copy the 6-digit verification code and paste it into the registration dialog box.
+   If you do not see the verification email, check your junk mail for an email from no-reply@verificationemail.com.
+
+   If you closed the registration dialog box, you can repeat Step 1 and then click on :guilabel:`Already have a code?`.
```
click on -> click

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+   Then enter your email and the verification code.
+
+You can now sign in on `nRF Cloud`_ with your email and the password your chose.
+After signing in, you are taken to the dashboard view that displays your device count and service usage.
```
Does this "taken to" sound a bit too concrete? I think the original "directed to" might be better or just "a dashboard is displayed".

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+
+#. Make a note of the ICCID and PUK codes.
+   You will need them later.
+   Pop out the nano-SIM card.
```
"Pop out..." should be a separate step from the taking a note step.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+#. Make a note of the ICCID and PUK codes.
+   You will need them later.
+   Pop out the nano-SIM card.
+   Take off the silicone cover on the Thingy:91, and put the card in the nano-SIM card holder.
```
Taking off the silicone cover should be a separate step.

mirjaku commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,571 @@
+#. In the :guilabel:`Device ID` field, enter the text ``nrf-`` and after it the Thingy:91 International Mobile (Station) Equipment Identity (IMEI) code.
+
+   The IMEI is the 15 digit code which you can find on a white sticker on the Thingy:91.
+   The Device ID is case sensitive, so make sure all the letters are lower-case.
```
The image showing where the PIN and IMEI are in the Thingy:91 is missing.

umapraseeda replied:
Added the image.

simensrostad commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,574 @@
+
+   .. note::
+
+      If you see an error message, check the error code and see `nRF Cloud REST API`_ in nRF Cloud to find out what is causing the error.
```
Is this the correct link? I don't really find any information that can help me debug the problem following that link.

simensrostad reviewed (approved):
The _Connecting the DK to nRF Cloud_ looks fine. Just a minor comment.

Avalei commented on doc/nrf/ug_thingy91.rst:
```diff
@@ -44,98 +38,55 @@ Thingy:91 uses the following UART baud rate configuration:
+#. Launch the LTE Link Monitor application.
+#. In the navigation bar, click :guilabel:`Select device`.
+   A drop-down menu appears.
+#. n the menu, select Thingy:91.
```
```suggestion
#. In the menu, select Thingy:91.
```

Avalei reviewed (changes requested):
## VSC Support
We should also include instructions on how to build and program with the VSC extension since it replaces SEGGER as the recommended tool. You could either reference our brief description on the [programming page](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-6561/nrf/gs_programming.html#building-with-the-vs-code-extension) or link directly to the [page on the VSC doc site](https://nrfconnect.github.io/vscode-nrf-connect/build_app.html). Please note that we are right in the middle of a VSC extension release that restructures that doc site content, so the direct url will break within the next few days and be replaced by a new one. Alternatively, you could duplicate the instructions and include them directly in the .rst page.

## Image Sizes
The max image size for NCS images is 800px x 780px. Many of the figures were too large in one or both dimensions. Can you re-export these within the recommended dimensions?

umapraseeda commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,574 @@
+
+         Programmer - Writing of HEX files
+
+      The :guilabel:`MCUboot DFU` window appears.
```
Added the screenshot for MCUboot DFU  window here as that is needed here.

umapraseeda commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,574 @@
+
+         Programmer - Update modem
+
+      The :guilabel:`Modem DFU via MCUboot` window appears.
```
Added the screenshot for Modem DFU via MCUboot window.

umapraseeda commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,574 @@
+
+   .. programmer_device_write_start
+
+   i. Click :guilabel:`Open`.
```
I had to add it manually as I was trying to reuse this set of statements later on. But I guess that messes up the bullet numbering. Fixed it now.

umapraseeda commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,574 @@
+
+         Programmer - Update modem
+
+      The :guilabel:`Modem DFU` window appears.
```
Added screenshot for Modem DFU window.

umapraseeda commented:
@Avalei, Thanks for the suggestions! I have added all the fixes and fixed all the screenshots. Also, I have added a section on building and programming using VS Code extension in the  'Working with Thingy' user guide.

umapraseeda commented on doc/nrf/ug_thingy91.rst:
```diff
@@ -11,22 +11,22 @@ Working with Thingy:91
-
-* nRF9160 SiP - supporting LTE-M, NB-IoT, and Global Navigation Satellite System (GNSS)
-* nRF52840 SoC - supporting Bluetooth® Low Energy and Near Field Communication (NFC)
+Thingy:91 integrates an nRF9160 SiP that supports LTE-M, NB-IoT, and Global Navigation Satellite System (GNSS) and an nRF52840 SoC that supports Bluetooth® Low Energy and Near Field Communication (NFC).
```
Added

crfosse reviewed (changes requested):
Just one note from me, otherwise it looks good.

umapraseeda commented on doc/nrf/ug_thingy91_gsg.rst:
```diff
@@ -0,0 +1,578 @@
+
+         Programmer - Select device
+
+   #. In the menu, select Thingy:91.
```
Rephrased the sentence after an offline discussion.

crfosse reviewed (approved):
LGTM
